h/static/scripts/tags.coffee

```diff
@@ -14,24 +14,24 @@ createTagHelpers = ['localstorage', (localstorage) ->
 
   # Add newly added tags from an annotation to the stored ones and refresh
   # timestamp for every tags used.
-  storeTags: (tags) ->
-    savedTags = localstorage.getObject TAGS_MAP_KEY
+  store: (tags) ->
+    savedTags = localStorage.getObject TAGS_MAP_KEY
     savedTags ?= {}
 
     for tag in tags
-      if savedTags[tag.text]?
+      if savedTags[tag]?
```

Comment:
I'll double check this, because I didn't think I changed anything that
would be incompatible. On the other hand, I'm not sure we've ever even
shipped this in production yet.
On Mar 26, 2015 11:01 AM, "Nick Stenning" <notifications@github.com> wrote:

> In h/static/scripts/tags.coffee
> <https://github.com/hypothesis/h/pull/2092#discussion_r27239009>:
>
> >      savedTags ?= {}
> >
> >      for tag in tags
> > -      if savedTags[tag.text]?
> > +      if savedTags[tag]?
>
> Ahhhh, sorry. I'd misread. Tags are guaranteed to just be strings now.
> That makes more sense. My point about backwards compatibility still stands,
> however.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hypothesis/h/pull/2092/files#r27239009>.
>